README.md

```diff
@@ -114,3 +114,32 @@ For adding new records the [adr-tools](https://github.com/npryce/adr-tools) comm
 ```bash
 brew install adr-tools
 ```
+
+## Contributing
+
+🇬🇧
+Everyone is welcome to contribute the development of the _<please_exchange_with_project_name>_. You can contribute by opening pull request,
```

Comment:
Replace `_<please_exchange_with_project_name>_` with `remix-application-template`?
`run.sh init` should automatically do the replacements to minimise manual work when applying the template.